src/main/java/stringaddcalculator/StringAddCalculator.java

```diff
@@ -0,0 +1,55 @@
+package stringaddcalculator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static int splitAndSum(String text){
+
+        if (text == null) return 0;
+
+        String delimiter = ",|:";
+
+        Map<String, String> delimiterAndTextMap = getCustomDelimiterAndText(delimiter, text);
+        delimiter = delimiterAndTextMap.get("delimiter");
+        text = delimiterAndTextMap.get("text");
+
+        if (text.equals("")) return 0;
```

Comment:
```suggestion
        if (text.isEmpty()) return 0;
```
빈 값이라면 `String`에서 제공하는 메소드를 사용할 수 있을 것 같아요 😄